skbeam/core/accumulators/tests/test_binned_statistic.py

```diff
@@ -6,7 +6,7 @@
 from numpy.testing import assert_array_equal, assert_array_almost_equal
 import numpy as np
 import scipy.stats
-from ...utils import bin_edges_to_centers
+from skbeam.core.utils import bin_edges_to_centers
```

Comment:
I personally don't think test files should do relative imports. Testing should occur from outside.

Reply:
would like an opinion from you @danielballan . @CJ-Wright  also suggested this should be a relative import, i agree more with him now as a test should be guaranteed to test the correct instance

Reply:
I think either one is ok.

Reply:
I decided to leave the global import finally. The tests are run in a controlled environment anyway so it shouldn't affect it. The bonus of global imports is you can run this as a script in ipython. Sometimes a little quicker for debugging, at least for me.
(I know there's the pytest --pdb option but sometimes I find it convenient to modify the script and have it return variables and play with them further, plot things etc.)